pyproject.toml

```diff
@@ -23,9 +23,8 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<4.0"
-importlib-metadata = {version = "^4.0", python = "<3.8"}
-ansys-fluent-core = "~=0.18.dev0"
+python = ">=3.9,<4.0"
+ansys-fluent-core = "~=0.18"
```

Comment:
Based on our discussion this is what I would recommend as a safer approach:
```suggestion
ansys-fluent-core = "~=0.18.0"
```
For:

![image](https://github.com/ansys/pyfluent-parametric/assets/132297401/2da007ac-0239-49ff-8dd7-f55babecb0ca)

So that when PyFluent 0.19 comes out (which is likely to include breaking changes) we need to do the update here manually. Alternatively, I think this is clearer with the same behavior as `"~=0.18"` as you mentioned (which it shouldn't have according to the PEP):
```suggestion
ansys-fluent-core = ">=0.18"
```

Reply:
Actually, we expect the parametric 0.9 will remain compatible with all future minor releases of pyfluent, as long as no braking change is introduced in pyfluent. If there is a breaking change in pyfluent, we will create a new minor release of parametric.

Reply:
Sounds good to me, and I think `>=0.18` is clearer and `~=0.18` is misleading. There is no reason to use `~=0.18` in this case, right? The two behave the exact same (but they should not according to the PEP), I couldn't find a single difference in behavior.

Reply:
`>=0.18` and `~=0.18` will be different for a major release (I also found this just now). Although we are not going to have any major release in pyfluent soon, I'll prefer keeping `~=0.18` to make the intention clear (compatible for all future minor releases).

Reply:
I see what you mean now, that is a very good point:
![image](https://github.com/ansys/pyfluent-parametric/assets/132297401/220dada8-0c57-44f7-a071-90abd7a630a7)